src/types/ITheme.ts

```diff
@@ -0,0 +1 @@
+export type ITheme = 'day' | 'night' | 'midnight';
```

Comment:
Hmn, Amino is an external library, so the type should not be named ITheme if the types are exported. When the consumer project imports it, they would expect ITheme to be an internal type of their project. Therefore, it would be more appropriate to use a different name for the type. For external library, type should be named without prefix `I`.